README.md

```diff
@@ -1 +1,91 @@
-# Recipe-App
\ No newline at end of file
+![Microverse](https://img.shields.io/badge/-Microverse-%20%236f23ff?style=for-the-badge) 
+
+![Ruby](https://img.shields.io/badge/ruby-%23CC342D.svg?style=for-the-badge&logo=ruby&logoColor=white) 
+![Rails](https://camo.githubusercontent.com/2f7aa65a22dedd331620c426fd36d88c01600b2c8ed750c41cc72b4d86364937/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f7261696c732d2532334343303030302e7376673f7374796c653d666f722d7468652d6261646765266c6f676f3d727562792d6f6e2d7261696c73266c6f676f436f6c6f723d7768697465)
+
+# Recipe App 
+
+> The Recipe app keeps track of all your recipes, and ingredients. You will be able to save ingredients, and keep track of what you have, create recipes. Also, the app allow you to make them public so anyone can access them. 
+
+
+## Built With :hammer_and_wrench: 
+
+- **Ruby:** 
+
+- **Rails:**  
+
+- **GitHub flow:**  
+
+- **Linters Check (Rubocop):** 
+
+- **PostgreSQL:** 
+
+## Entity Relationship Diagram 
+
+![](./recipe_erd.png)
+
+## Getting Started ➡️
+
+To get a local copy up and running follow these simple steps.:arrow_down:
+
+### Prerequisites ✅
+- A code editor like Visual Studio Code with Git and Ruby.
+- You can check if Git is installed by running the following command in the terminal: 
+
+```
+$ git --version
+```
+
+Likewise for Ruby installation.
+
+```
+$ ruby --version
+``` 
+
+### Setup
+
+1. Clone the GitHub repository using the  this link 
+```
+  $ git clone https://github.com/renercrows/Recipe_App.git
```

Comment:
- [ ] After cloning your application, creating the database and running migrations, the process i canceled because of some duplication in your database schema. See screenshot below:
![Screenshot from 2022-12-03 07-19-51](https://user-images.githubusercontent.com/62849872/205427841-ffc1131e-198d-4e07-a206-fb1bd7977fd3.png)
- See also the screenshot below of the error I get when I start the server. All this is because of the duplication as indicated in the screenshot above. Kindly fix the duplication in the schema.
![Screenshot from 2022-12-03 07-24-23](https://user-images.githubusercontent.com/62849872/205427930-b3fccca1-7e4f-4885-9c6b-bda731551447.png)

Reply:
Thank you, already fixed it.